### PR TITLE
Don't spit out traceback when CLI parameters are missing

### PR DESCRIPTION
**Pull request by hoechenberger**
`main`:
```
❯ mne_bids_pipeline
Traceback (most recent call last):
  File "/Users/hoechenberger/mambaforge/envs/mne/bin/mne_bids_pipeline", line 8, in <module>
    sys.exit(main())
  File "/Users/hoechenberger/Development/mne-bids-pipeline/mne_bids_pipeline/run.py", line 71, in main
    raise ValueError(bad_msg)
ValueError: You must specify a configuration file as a single argument or with --config.
```
PR:
```
❯ mne_bids_pipeline
[2022-11-09 22:14:59] │ 😵  You must specify a configuration file as a single argument or with --config.
```

### Before merging …

- [ ] Changelog has been updated (`docs/source/changes.md`)

### CODE REVIEW

File: mne_bids_pipeline/_logging.py [hunk 1]
```diff
@@ -69,7 +69,6 @@ def gen_log_kwargs(
     session: Optional[Union[str, int]] = None,
     run: Optional[Union[str, int]] = None,
     emoji: str = '⏳️',
-    script_path: Optional[PathLike] = None,
```
Comment (hoechenberger) [inline mne_bids_pipeline/_logging.py:72]: This parameter was unused, so I removed it

File: mne_bids_pipeline/_logging.py [hunk 2]
```diff
@@ -80,8 +79,12 @@ def gen_log_kwargs(
 
     message = f' {message}'
 
-    script_path = pathlib.Path(os.environ['MNE_BIDS_STUDY_SCRIPT_PATH'])
-    step_name = f'{script_path.parent.name}/{script_path.stem}'
+    script_path = os.environ.get('MNE_BIDS_STUDY_SCRIPT_PATH')
```
Comment (hoechenberger) [inline mne_bids_pipeline/_logging.py:82]: The env var gets set once we're running a processing script; it however run.py wants to log a message, this variable is not set, so we have to handle this

File: pyproject.toml [hunk 3]
```diff
@@ -66,7 +66,7 @@ tests = [
 ]
 
 [project.scripts]
-mne_bids_pipeline = "mne_bids_pipeline.run:main"
+mne_bids_pipeline = "mne_bids_pipeline.run:main_cli"
```
Comment (hoechenberger) [inline pyproject.toml:69]: the new behavior only affects the CLI; if instead one programmatically calls `run.main()`, everything remains unchanged and one gets the full traceback

**Comment by hoechenberger** [comment]
@larsoner WDYT?

**Comment by hoechenberger** [comment]
The only annoying thing is the **two** spaces after the emoji!

**Comment by larsoner** [comment]
I think the standard behavior here might be to show the equivalent of `--help` when no arguments or switches are provided. But even in that case if they do something like `mne_bids_pipeline --debug` maybe we want this message, not sure. So I think this is okay/explicit enough

**Comment by hoechenberger** [comment]
I'm not sure I fully understand – are you okay with these changes?

File: pyproject.toml (hunk 3 above)
Comment (larsoner) [reply pyproject.toml:69]: I think in practice nobody will programatically call `run.main()` (and it should be private).

Have you checked to make sure `run_tests.py` with `--debug` for example give reasonable outputs? And with `on_error = 'abort'` this doesn't now print redundant information or anything? If so then I'm okay with this

**Comment by hoechenberger** [comment]
@larsoner I've checked and the behavior for everything happening inside the processing scripts is unchanged; this here only affects errors raised in what was previously called `run.py` -- which I have now renamed to `_main.py`

**Pull request merged**